src/app/dashboard/Missions/MapView.jsx

```diff
@@ -56,31 +56,46 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const Overview = ({ missions }) => {
+const Overview = ({ missions, selectedMission }) => {
   const classes = useStyles();
 
-  const FastFoodIconHtml = renderToString(<FastfoodIcon />);
+  const position = { lat: 37.773972, lng: -122.431297 };
```

Comment:
This will be calculated at some point yeh?

Reply:
yeah, we need this data from organization lat Lng...